Susi/Controllers/LoginController/LoginVCMethods.swift

```diff
@@ -256,14 +256,6 @@ extension LoginViewController: UITextFieldDelegate {
         passwordTextField.text = ""
     }
 
-    // Show sign up controller
-    func presentSignUpController() {
-        clearFields()
-
-        UserDefaults.standard.setValue(emailTextField.text, forKey: ControllerConstants.UserDefaultsKeys.typedEmailAdress)
```

Comment:
I think we are using this UserDefault value to pass the email address

Reply:
But, this function is not referred or called anywhere.

Reply:
Okay!